plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryCliProvider.kt

```diff
@@ -27,7 +27,9 @@ internal object SentryCliProvider {
         project.logger.info("[sentry] cliSuffix is $cliSuffix")
 
         if (!cliSuffix.isNullOrBlank()) {
-            val resourcePath = "/bin/sentry-cli-$cliSuffix"
+            val sep = File.separator
+
+            val resourcePath = "{$sep}bin{$sep}sentry-cli-$cliSuffix"
```

Comment:
@cortinico wondering if it should be `File.separator` for the java resources or if it's OS independent.
most tests are failing, but the test for this use case does use `File.separator`, ideas? ^^

Reply:
I believe we should migrate the whole thing to use `kotlin.io.path.Path`. It's experimental in Kotlin 1.4 but allows us to get rid of these kind of problems. As an alternative we could use `Path` from nio directly.

Reply:
yeah, makes sense, but I guess it's failing because the Java resource path is OS independent, maybe?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#getResource(java.lang.String)

`getResource` requires `/`, so I am just removing `File.separator` from the test instead of fixing it here